src/main.cpp

```diff
@@ -408,13 +408,13 @@ auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpo
 	}
 } // load_oidc_configuration
 
-class bearer_token_eviction_manager
+class process_stash_eviction_manager
```

Comment:
I don't understand. Please elaborate.

Reply:
I don't think we need to worry about refactoring the eviction manager class. It probably won't see any additional changes after this PR is merged because it only does one thing. That being, walk the process stash and delete expired data.

_This is my thought after @alanking explained what he meant offline._